menas/src/main/scala/za/co/absa/enceladus/menas/controllers/VersionedModelController.scala

```diff
@@ -69,6 +69,12 @@ abstract class VersionedModelController[C <: VersionedModel with Product with Au
     }
   }
 
+  @GetMapping(Array("/detail/{name}/latestVersion"))
+  @ResponseStatus(HttpStatus.OK)
+  def getLatestVersionNumber(@PathVariable name: String): CompletableFuture[Int] = {
```

Comment:
Let's get an API test for this. (see `za.co.absa.enceladus.menas.integration.controllers.SchemaApiIntegrationSuite:43` for inspiration.

Reply:
Have you seen the line 98? Seems like we will have to have the second issue opened to handle it with rest as well.